Automattic/wp-calypso
# Make password view when email is present

Related to #53495

## Proposed Changes

When loading the login form, checks if there's email or username filled, if so, load the form with password field visible.
This is helpful in this https://github.com/Automattic/wp-calypso/issues/53495 

## Testing Instructions



* Pull and run this branch or use calypso live link
* Navigate using incognito or logged-out user to /log-in
* Fill your email or username and click in continue
* In the next step, click in "Email a login link"
* Click on “enter a password instead”
* You should see the login for with the password field enabled.

## Pre-merge Checklist
